sql/shredder_state/progress/view.sql

```diff
@@ -0,0 +1,147 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shredder.shredder_state.progress`
+AS
+WITH max_end_date AS (
+  SELECT
+    MAX(end_date) AS end_date
+  FROM
+    `moz-fx-data-shredder.shredder_state.shredder_state`
+),
+shredder_state AS (
+  SELECT
+    task_id,
+    -- NEWEST job for each task_id
+    ARRAY_AGG(
+      STRUCT(
+        start_date,
+        end_date,
+        SPLIT(job_id, ":")[offset(0)] AS project_id,
+        SPLIT(job_id, ".")[offset(1)] AS job_id
+      )
+      ORDER BY
+        job_created DESC
+      LIMIT
+        1
+    )[OFFSET(0)].*
+  FROM
+    `moz-fx-data-shredder.shredder_state.shredder_state`
+  -- only for the most recent shredder run
+  JOIN
+    max_end_date
+  USING
+    (end_date)
+  GROUP BY
+    task_id
+),
+tasks AS (
+  SELECT
+    task_id,
+    ARRAY_AGG(
+      STRUCT(start_date, end_date, target, target_bytes, source_bytes)
+      -- already filtered on max_end_date, so use OLDEST metadata
+      ORDER BY
+        _PARTITIONTIME
+      LIMIT
+        1
+    )[OFFSET(0)].*
+  FROM
+    `moz-fx-data-shredder.shredder_state.tasks`
+  JOIN
+    max_end_date
+  USING
+    (end_date)
+  GROUP BY
+    task_id
+),
+jobs AS (
```

Comment:
Perhaps a link to https://cloud.google.com/bigquery/docs/information-schema-jobs around here, or the particular reference you used to develop this query.

Reply:
that's the only reference i used.